src/js/lib/client/src/index.js

```diff
@@ -1,14 +1,15 @@
 'use strict'
 
-import * as buildCfg  from '../../../../../dist/build.json'
 import * as Electron  from 'electron'
+import * as Server  from 'enso-studio-common/src/server'
+import * as assert from 'assert'
+import * as buildCfg from '../../../../../dist/build.json'
```

Comment:
sort

Reply:
According to my sort plugin, they are sorted. What would be the preferred sorting?